justfile

```diff
@@ -223,6 +230,11 @@ wait-for-electrs-to-be-ready:
     done
 
 build-ipa:
+    #!/usr/bin/env bash
+    ELECTRS_ENDPOINT="${ELECTRS_ENDPOINT:-$TESTNET_ELECTRS}"
+    COORDINATOR_P2P_ENDPOINT="${COORDINATOR_P2P_ENDPOINT:-$TESTNET_COORDINATOR}"
```

Comment:
are you sure this does what you wanted it to do?
maybe something more like:
`ELECTRS_ENDPOINT="${ELECTRS_ENDPOINT}:-${TESTNET_ELECTRS}" ? (quoting the individual variables?)

Reply:
Am I misunderstanding how to set a default if the variable is not set? 
I followed this: https://stackoverflow.com/questions/2013547/assigning-default-values-to-shell-variables-with-a-single-command-in-bash

Reply:
please maybe add a comment what's happening here - I've been doing Bash for years and thought I'm quite decent, and you totally blew my mind with this! 😂

thank you for enlightening me!